main.go

```diff
@@ -54,6 +54,13 @@ func main() {
 	RootCmd.PersistentFlags().StringVarP(&globalOpts.Repo, "repo", "r", "", "Repository directory to backup to/restore from (default: current working dir)")
 	RootCmd.PersistentFlags().StringVarP(&globalOpts.Password, "password", "p", "", "Password to use for data encryption")
 
+	if globalOpts.Repo == "" {
```

Comment:
Are we sure the flags have already been parsed and set at that point? If my memory serves me right, this only happens when executing the command (`RootCmd.Execute`), so `globalOpts.Repo` would always be empty here. Maybe I'm wrong, maybe that has changed by now. Have you checked?